CSharpHacks/CSharpHacks.Tests/EnumerableExtensionsTests.cs

```diff
@@ -111,6 +111,30 @@ public void Omit_should_return_empty_enumerable_if_number_is_larger_than_size_of
             Assert.Equal(expected, test.Omit(1000));
         }
 
+        [Fact]
+        public void Omit_should_be_affected_by_mutating_source()
+        {
+            int[] expected = { 0, 2, 3, 4, 5, 6 };
```

Comment:
please could you change these variable definitions to use "var" instead of the definite type?